rasterfoundry/settings.py

```diff
@@ -1,4 +1,4 @@
 RV_CPU_QUEUE = 'raster-vision-cpu'
 RV_CPU_JOB_DEF = 'raster-vision-cpu'
-RV_PROJ_CONFIG_DIR_URI = 's3://raster-vision-od/configs/projects'
+RV_PROJ_CONFIG_DIR_URI = 's3://raster-vision-lf-dev/detection/configs/projects/rf-generated'
```

Comment:
shouldn't these kinds of settings be stored in a gitignored file with an example placeholder file committed? We do something similar in RF

Reply:
Good point. I have too much other stuff do right now, so I made an issue to do that: https://github.com/raster-foundry/raster-foundry-python-client/issues/43